### ISSUE TITLE
eval bad

### ISSUE THREAD

**Issue by deavn2236** [MarkFzp/act-plus-plus#18]
When I had the following warning in training, and finally eval through imitate_episodes is also bad, what is the reason
Warning: step duration: 0.091 s at step 211 longer than DT: 0.02 s, culmulated delay: 17.275 s

**Comment by EndlessLoops** [comment]
Are you fixed it ?

**Comment by nicklhy** [comment]
Same problem here.

**Comment by AbdElrahmanMostafaRifaat1432** [comment]
increase batch size from 8 to 16 and increase learning rate to  2e-5 or 1e-4  and increase number of steps to more than 6000 at least you will achieve 50 % if you want more increase number of steps
